### DISCUSSION TITLE
Can't start superset-frontend project(Nullish coalescing operator) "??"

### DISCUSSION THREAD

**Discussion by ByePastHub** [apache/superset#18315] [Q&A / Help]
Can't start superset-frontend project(Nullish coalescing operator)

### Expected results

```
|   const initialState = {
>     pageSize: initialPageSize ?? DEFAULT_PAGE_SIZE,
|     pageIndex: initialPageIndex ?? 0,
|     sortBy: initialSortBy };
```

### Actual results

Compilation failed

#### Screenshots

<img width="1358" alt="iShot2021-07-21 17 08 01" src="https://user-images.githubusercontent.com/40738560/126463166-b74970ee-9823-4a30-9af1-e4facb051f6c.png">

#### How to reproduce the bug

1. git clone project(master)
2. In the superset-frontend directory
3. yarn install
4. npm run dev

### Environment

(please complete the following information):

- superset version: `I don't know what version it is，2021-7-21 master branch`
- python version: `2.7.16`
- node.js version: `v14.16.0`
- platform `macOS Big Sur 11.4`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by ByePastHub** [comment]
Does anyone have the same problem as me?

**Comment by piewaer** [comment]
I also encountered exactly the same mistakes. After a round of searching, I didn't find a solution, and no one answered my questions. It's not just?? Moreover, JSX can't be parsed. I checked webpack.config.js and babel.config.js and found no configuration problems. I want to cry

Run the program with docker, you will report errors, but you will ignore them, and finally you can run

**Comment by stephenLYZ** [comment]
@ByePastHub Hi,  I hope this  [CONTRIBUTING.md](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#frontend).will help you to start your local environment.

**Comment by ByePastHub** [comment]
> @ByePastHub Hi, I hope this [CONTRIBUTING.md](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#frontend).will help you to start your local environment.

I did follow this, but this problem still occurs. Not only my computer has this problem, but my colleague’s computer also has this problem, but some of my colleagues are very strange and don’t appear.

**Comment by ByePastHub** [comment]
![code](https://user-images.githubusercontent.com/40738560/127119579-9a472745-ee5f-4119-9586-6451375d4b6c.PNG)

Windows 10 will also have problems, which is a bit different from the one above.

superset version: v2021.25.2(tag)
node version: v14.16.0

**Comment by RickYangzz** [comment]
To use 'npm ci' add dependencies. Don't use 'npm install'.

**Comment by nikhil-kuyya-talentas** [comment]
I guess you need to use npm instead of yarn in step3.
